dataprocess/dataprocess.go

```diff
@@ -0,0 +1,38 @@
+// Package dataprocess manages code related to the data-accepting process
+package dataprocess
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"os"
+
+	pc "go.viam.com/rdk/pointcloud"
+)
+
+// WritePCDToFile Encodes the pointcloud and then saves it to the passed filename.
+func WritePCDToFile(ctx context.Context, pointcloud pc.PointCloud, filename string) error {
```

Comment:
[nit] We need to pass `context.Context` only when it is used within the function (or passed along into other functions that use it).

```suggestion
func WritePCDToFile(pointcloud pc.PointCloud, filename string) error {
```